Resource.js

```diff
@@ -129,12 +135,20 @@ module.exports = function(app, route, modelName, model) {
               })
             });
             break;
+          case 204:
+            // Convert 204 into 200, to preserve the empty result set.
+            if (req.method !== 'DELETE') {
```

Comment:
Is this going to change ALL responses to an empty array? POST, PUT and some GETs are expecting an object so we should return an empty object instead of an empty array unless an array is expected.